deploy/helm/magda/charts/registry-api/templates/deployment.yaml

```diff
@@ -50,7 +50,7 @@ spec:
             port: 80
           initialDelaySeconds: 60
           periodSeconds: 10
-          timeoutSeconds: 10
+          timeoutSeconds: 3
```

Comment:
What's the reason we lower the timeout value?
Asked because of I quite often have the situation (in my local minikube) that registry API is killed due to high CPU usage. And the killing & re-creation action itself add more burden to the node and causing other pods starting response slower (which may trigger more pods being killed).

Reply:
Yeah good question. The answer is that it mitigates #749 - right now sometimes the registry gets into a state where it uses every CPU cycle it can but doesn't actually process any webhooks... it still responds to API requests within 10 seconds, but when these requests com from outside the network they end up being 15 seconds + :(.

So for now I've set a really tight `timeoutSeconds` value to ensure it's rebooted as soon as it slows down.

In the gitlab PR I disabled liveness probes except in prod but you have a good point in that this is going to make life miserable on minikube until that gets merged... I'll make this prod only.